cache/edge_lambdas/src/toggler.ts

```diff
@@ -19,7 +19,14 @@ type Test = {
 };
 
 // This is mutable for testing
-export let tests: Test[] = [];
+export let tests: Test[] = [
```

Comment:
I don't think I'm able to import this from the edge_lambda into the toggles directory and as such this duplication is necessary. Although I'm not 100% sure why that is the case, and would be happy to be shown that it wasn't.